### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]: Keyboard focus is not visible for "Available packs" to install in "Install Details" Window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1352]
**Repro Steps:**
1. Open and Start Eclipse.
2.  Press "Alt+H" to open Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter
5. Then select "Azure Toolkit" checkbox and Tab  up to "Contact all update sites during install to find required software" checkbox then press tab to "Next" button and hit enter.
6. In Install details dialog, press tab to review the items to be installed and press tab then observe the keyboard focus

**Expected Result:**
Keyboard focus should be visible for available packs to install present under Name.

**Actual Result:**
Keyboard focus is not visible for available packs to install present under Name
**Observation:** keyboard focus is not visible for available packs, but by using arrow keys we are able to access the available packs. Once user traverse the available packs we are able to see the focus for the next time navigation

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(OS Build 16299.192)

![keyboard focus is not visible for highlighted pane in install details](https://user-images.githubusercontent.com/22956130/35389822-2c2b9dde-01ff-11e8-9119-c5866e0830b0.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a)
OS Version - 1803 (OS Build 17134.167)
 .Issue Still Repro .
![focus visible-install details](https://user-images.githubusercontent.com/22956130/43827592-d993b4fa-9b17-11e8-9c44-54064cd68428.gif)

**Comment by jdneo** [comment]
@WiproPSAccTest Can you try to use up/down arrow to switch the focus?

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. A simple workaround is available, users can install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749827
